models/demos/t3000/falcon40b/tt/falcon_decoder.py

```diff
@@ -415,36 +364,29 @@ def fwd_decode(
         )
         attention_output, outputs = attn_outputs[0], attn_outputs[1:]
 
-        output = []
-
         # Add attn output to residiual first in place to save memory
         # Note that this is only correct in inference when dropout is disabled
-        for i in range(len(residual)):
-            output.append(
-                ttnn.add(
-                    residual[i],
-                    attention_output[i],
-                    memory_config=self.model_config["PARALLEL_ATTN_ADD_OUTPUT_MEMCFG"],
-                    output_tensor=residual[i],
-                )
-            )
-            attention_output[i].deallocate(True)
+        output = ttnn.add(
+            output,
+            attention_output,
+            memory_config=self.model_config["PARALLEL_ATTN_ADD_OUTPUT_MEMCFG"],
+            # output_tensor=output,
+        )
+        attention_output.deallocate(True)
 
         # MLP
         # mlp will deallocate layernorm_output
         mlp_output = self.mlp(mlp_ln_output, llm_mode=llm_mode)
 
         # dropout_add
         # For inference, this is just add
-        for i in range(len(output)):
-            output[i] = ttnn.add(
-                output[i],
-                mlp_output[i],
-                memory_config=self.model_config["DROPOUT_ADD_OUTPUT_MEMCFG"],
-                output_tensor=output[i],
-            )
-
-            mlp_output[i].deallocate(True)
+        output = ttnn.add(
+            output,
+            mlp_output,
+            memory_config=self.model_config["DROPOUT_ADD_OUTPUT_MEMCFG"],
+            # output_tensor=output,
```

Comment:
remove and please check everywhere for commented code

Reply:
Same as above